pkg/alertmanager/amcfg.go

```diff
@@ -1140,6 +1140,25 @@ func (cg *configGenerator) convertHTTPConfig(ctx context.Context, in monitoringv
 		out.BearerToken = bearerToken
 	}
 
+	if in.OAuth2 != nil {
```

Comment:
we need a version check in `sanitize()` since OAuth2 is supported with >= 0.22.0

Reply:
ack..missed that